srcpkgs/libjpeg-turbo/patches/0001-tjLoadImage-Fix-int-overflow-segfault-w-big-BMP.patch

```diff
@@ -0,0 +1,71 @@
+From 1e18a1a09af9f143400cedc54a210f616c80ffb9 Mon Sep 17 00:00:00 2001
+From: DRC <information@libjpeg-turbo.org>
+Date: Tue, 1 Jan 2019 18:57:36 -0600
+Subject: [PATCH] tjLoadImage(): Fix int overflow/segfault w/big BMP
+
+Fixes #304
+---
+ ChangeLog.md | 17 +++++++++++++++++
+ turbojpeg.c  |  9 ++++++---
+ 2 files changed, 23 insertions(+), 3 deletions(-)
+
+diff --git ChangeLog.md ChangeLog.md
```

Comment:
changes to ChangeLog.md can be excluded to make the patches smaller

Reply:
@maxice8 Fixed